Packs/Code42/Integrations/Code42/README.md

```diff
@@ -314,6 +314,54 @@ Get all employees on the Departing Employee List.
 |  | leaving for competition | 951984198921509692 | user7@example.com.com |
 |  | Leaving for competitor | 895005723650937319 | user8@example.com |
 
+### code42-departingemployee-get
+***
+Fetch user details from the Departing Employee List.
+
+#### Base Command
+
+`code42-departingemployee-get`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| username | The username of the Departing Employee. | Required |
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| Code42.DepartingEmployee.TenantID | string | Internal Code42 tenant ID for the Departing Employee. |
+| Code42.DepartingEmployee.Username | string | The username of the Departing Employee. |
+| Code42.DepartingEmployee.DisplayName | string | Display name associated to the Departing Employee. |
+| Code42.DepartingEmployee.Notes | string | Notes associated to the Departing Employee. |
+| Code42.DepartingEmployee.CreatedAt | unknown | Timestamp at which the employee was added to Departing Employee List. |
+| Code42.DepartingEmployee.Status | string | Status associated to the Departing Employee. |
+| Code42.DepartingEmployee.CloudUsernames | unknown | Cloud user IDs associated to the Departing Employee. |
+| Code42.DepartingEmployee.Type | string | Internal Code42 API version number. |
+
+
+#### Command Example
+```!code42-departingemployee-get username="partner.demisto@example.com```
+
+#### Context Example
+```
+{
+    "Code42": {
+        "DepartingEmployee": {
```

Comment:
To confirm, as I wasn't able to test this. Response doesn't contain the keys `code42` & `DepartingEmployee` but I assume it must be getting added while making response adhere to Demisto format. Hence documented alike other commands.

Reply:
Once you get your environment working, you should be able to automatically create the docs using `demisto-sdk generate-docs`.